apps/web/components/getting-started/steps-views/UserSettings.tsx

```diff
@@ -69,7 +70,7 @@ const UserSettings = (props: IUserSettingsProps) => {
     <form onSubmit={onSubmit}>
       <div className="space-y-6">
         {/* Username textfield */}
-        <UsernameAvailabilityField user={user} />
+        <UsernameAvailabilityField user={user} organization={organization} />
```

Comment:
My two cents here. Let's use trpc inside to avoid prop drilling. We can take advantage of suspense here as well.

```ts
const [user] = trpc.viewer.me.useSuspenseQuery();
```